### ISSUE TITLE
Does conv2D transpose converting work properly?

### ISSUE THREAD

**Issue by mrlzla** [nerox8664/pytorch2keras#7]
Should we simply add zeropadding before convolution transpose or we should use different advanced feature? :)

**Comment by nerox8664** [comment]
Hello @mrlzla.
Yep, there are some problems with `stride`. As I can see, it works different in PyTorch and in Keras(TF). Simple add zeropadding before convolution isn't a good decision because of ConvTranspose2d mechanics.
You can check it [in that visualization](https://github.com/vdumoulin/conv_arithmetic).

**Comment by mrlzla** [comment]
Do you know if there some way to fix it?
Probably instead of adding some padding we should crop center with appropriate size?))

**Comment by mrlzla** [comment]
Yes, I used Cropping2D with cropping=(node.padding[0], node.padding[1]) after Conv2DTranspose and it is  partially solved the problem. It works well in my case but some tests fail.
